packages/core/lib/services/dock/service.js

```diff
@@ -68,7 +68,7 @@ export class DockService {
 
     this._setDockReady(true);
 
-    return result;
+    return true;
```

Comment:
It was creating a memory leak, as the sdk connection will be parsed to JSON by the rpc server.